util/pom.xml

```diff
@@ -96,6 +96,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>pl.pragmatists</groupId>
+      <artifactId>JUnitParams</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I don't like to introduce a new testing framework only for this one case. I would prefer to migrate to JUnit 5 instead and use a more plain test style before.